Doc/library/fractions.rst

```diff
@@ -89,6 +90,9 @@ another rational number, or from a string.
       and *denominator*. :func:`math.gcd` always return a :class:`int` type.
       Previously, the GCD type depended on *numerator* and *denominator*.
 
+   .. versionchanged:: 3.11
+      Supported :PEP:`515`-style initialization from string.
```

Comment:
Could we reword or expand this to explicitly mention underscores? (Else someone reading the docs needs to either know what PEP 515 is, or follow the link to find out.) Maybe something along the lines of "Underscores are now permitted when creating a :class:`Fraction` instance from a string, following :PEP:`515` rules."